ibm_catalog.json

```diff
@@ -78,6 +78,67 @@
                 },
                 "description": "This architecture supports creating and configuring IBM Secrets Manager instance."
               }
+            ],
+            "configuration": [
+              {
+                  "key": "cos_instance_name",
+                  "type": "string",
+                  "default_value": "base-security-services-cos",
+                  "description": "The name to use when creating the Cloud Object Storage instance.",
+                  "required": false
+              },
+              {
+                "key": "region",
+                "type": "string",
+                "required": true,
+                "default_value": "",
+                "options": [
```

Comment:
This list should probably be in alphabetical order as I think its an ordered list. Its also missing these supported regions:

- Sao Paulo
- Toronto
- BNPP_EU (do we want to include this or not?)
- Washington DC